testing/integration/agent_long_running_leak_test.go

```diff
@@ -154,7 +154,7 @@ func (runner *ExtendedRunner) TestHandleLeak() {
 	timer := time.NewTimer(testDuration)
 	defer timer.Stop()
 
-	ticker := time.NewTicker(time.Second * 10)
+	ticker := time.NewTicker(time.Second * 60)
```

Comment:
@pchila From the [issue](https://github.com/elastic/elastic-agent/issues/5279) linked in PR description,

> TestLongRunningAgentForLeaks is failing due to following reason:
> 
> Flaky for linux/windows
> It fails due to the monitoring module staying in a degraded mode for a while, it will eventually return to a healthy state. It sometimes takes a few dozen seconds for libbeat to start the metrics endpoint and we only wait for 10 seconds.
> FIX: Waiting for 60s should solve this
>